mods/ts/installer/downloads.yaml

```diff
@@ -15,7 +15,202 @@ basefiles: Base Freeware Content
 		^Content/ts/speech02.mix: speech02.mix
 		^Content/ts/tem.mix: tem.mix
 		^Content/ts/temperat.mix: temperat.mix
-music: Freeware Music
+
+tibsun-music: Base Freeware Music
 	MirrorList: http://www.openra.net/packages/ts-music-mirrors.txt
 	Extract:
 		^Content/ts/scores.mix: scores.mix
+
+fstorm: Expansion Freeware Content
+	URL: http://81.169.246.181/openra/ts/ts-expand.zip
```

Comment:
Can you please add these to https://github.com/OpenRA/OpenRAWeb/pull/316?  You'll need to create a new mirror txt file. Once that is merged we can set these properly.